Makefile

```diff
@@ -70,7 +70,9 @@ $(GOBINDATA_BIN):
 	$(GO) build -o $(GOBINDATA_BIN) ./vendor/github.com/kevinburke/go-bindata/go-bindata
 
 test-e2e: $(BINDATA)
-	KUBERNETES_CONFIG="$(KUBECONFIG)" $(GO) test -v -timeout 20m -tags e2e ./test/e2e/... -run .
+	for d in basic reboots; do \
```

Comment:
may i recommend a script running the suits recursively and add color setting selectively? 
see https://github.com/openshift-kni/performance-addon-operators/blob/master/hack/run-functests.sh

Reply:
I can see some value, however adding a `hack` directory with random scripts is something I managed to avoid so far.  I'd also like to avoid pulling extra dependencies like `github.com/onsi/ginkgo/ginkgo` which are not necessarily needed.  If they really are needed, they'd have to be vendored in as from some build environments you cannot fetch external dependencies.  As for vendoring in, I tend to follow "vendor in only things that are essential".  Can you elaborate whether that dependency is essential?

As for running tests recursively, I wanted to enforce the ordering: basic tests first, advanced (requiring reboots) next.  Also having the flexibility to enable/disable tests seemed handy.